src/main/kotlin/org/jetbrains/research/tasktracker/ui/main/panel/MainPluginPanelFactory.kt

```diff
@@ -5,36 +5,42 @@ import com.intellij.openapi.wm.ToolWindow
 import com.intellij.openapi.wm.ToolWindowFactory
 import com.intellij.ui.components.JBPanel
 import com.intellij.ui.jcef.JBCefApp
-import java.awt.GridLayout
+import org.jetbrains.research.tasktracker.ui.main.panel.template.TasksPageTemplate
+import java.awt.BorderLayout
+import java.awt.FlowLayout
 import java.awt.event.ActionListener
 import javax.swing.JButton
 
 class MainPluginPanelFactory : ToolWindowFactory {
-    private val button = JButton("click me")
+    private val button = JButton("next")
+    private lateinit var mainWindow: MainPluginWindow
 
     override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
-        val mainWindow = project.getService(MainWindowService::class.java).mainWindow
+        mainWindow = project.getService(MainWindowService::class.java).mainWindow
         val jComponent = toolWindow.component
-        val panel = JBPanel<JBPanel<*>>(GridLayout(2, 1)).apply { // TODO need we GridLayout?
-            add(mainWindow.jComponent)
-            add(
-                JBPanel<JBPanel<*>>().apply {
-                    add(
-                        button.apply {
-                            addListener {
-                                mainWindow.loadDefaultPage("tasks")
-                                // TODO refactor, add function for button listener and text change
-                            }
-                        }
-                    )
-                }
-            )
+        button.apply {
+            isBorderPainted = false
+            addListener {
+                selectTask()
+            }
         }
-        jComponent.parent.add(panel)
+        val buttonPanel = JBPanel<JBPanel<*>>(FlowLayout()).apply {
+            add(button)
+        }
+        val panel = JBPanel<JBPanel<*>>(BorderLayout()).apply {
+            add(mainWindow.jComponent, BorderLayout.NORTH)
+            add(buttonPanel)
+        }
+        jComponent.add(panel)
     }
 
     override fun isApplicable(project: Project) = super.isApplicable(project) && JBCefApp.isSupported()
 
+    private fun selectTask() {
+        mainWindow.loadDefaultPage(TasksPageTemplate)
+        button.text = "select"
```

Comment:
It would be great to add a button "back"

<img width="492" alt="Screenshot 2023-07-20 at 14 45 24" src="https://github.com/JetBrains-Research/tasktracker-3/assets/8993915/cfc5d16d-ab70-45a6-a7cb-7b5076940545">

Reply:
Could you also put default languages here?

Reply:
Currently we haven't languages in config files, I'll add it in 'hello world' task